K8s/soda_scale_service.sh

```diff
@@ -0,0 +1,17 @@
+#!/bin/sh
+no_of_pods="$1"
+service_name="$2"
+
+if [ -z "$service_name" ] || [ -z "$no_of_pods" ]
+then
+        echo "Usage : ./soda_scale_service <service_name> <number_of_instances>"
```

Comment:
should there be more info for the placement of soda_scale_service? './' will always mean current dir

Reply:
its experimental for this release . hence not exposed out.

Reply:
Done